pytorch_lightning/profilers/__init__.py

```diff
@@ -0,0 +1,15 @@
+from pytorch_lightning.profilers.advanced import AdvancedProfiler
```

Comment:
can you add the license from another module here?

Reply:
sure